enhancements/network/static-ip-addresses-vsphere.md

```diff
@@ -0,0 +1,565 @@
+---
+title: static-ip-addresses-vsphere
+authors:
+  - rvanderp3
+reviewers: 
+  - JoelSpeed
+  - elmiko 
+  - patrickdillon
+  - jcpowermac
+  - cybertron
+  - zaneb
```

Comment:
Each person requested here is requested for a subject area right? Can you please explain which subject areas they are requested for (a habit we've been trying to promote recently)

Reply:
for sure, I can do that.